site/pom.xml

```diff
@@ -125,6 +125,11 @@
             <artifactId>junit-dataprovider</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.splunk.logging</groupId>
+            <artifactId>splunk-library-javalogging</artifactId>
+            <version>1.5.2</version>
```

Comment:
Seeing that the `splunk-library-javalogging` has already been defined in the parent POM's `<dependencyManagement>` section (:+1: ), the declaration in this POM file doesn't need `<version>` specified. 

Te idea behind `<dependencyManagement>` is that it defines a pool of common 'blessed' dependencies for child POMs to choose from in which case they only need to specify `groupId` and `artifactId` themselves.

Reply:
Thanks, version removed.